include/ibamr/FEMechanicsBase.h

```diff
@@ -480,6 +530,11 @@ class FEMechanicsBase : public SAMRAI::tbox::Serializable
                                                  unsigned int part,
                                                  const std::string& extension);
 
+    /*!
+     * Cached input databases.
+     */
+    SAMRAI::tbox::Pointer<SAMRAI::tbox::Database> d_fe_data_manager_db, d_fe_projector_db;
```

Comment:
`FEMechanicsBase` doesn't know anything about `FEDataManager` (nor should it). We should ignore any appearances of "FEDataManager" in the input database to this class (`IBFEMethod` can deal with it itself later). Since the idea of passing an `FEProjector` database is new we can fix the problem without worrying about backwards compatibility. Lets just check if `FEProjector` is a database in the input database to this class - if it is we pass it along, if not we create a new empty database and store that.

Reply:
I have removed `d_fe_data_manager_db` from `FEMechanicsBase` and moved the checks for those input settings to `IBFEMethod`.